transport/http/handler.go

```diff
@@ -82,6 +88,26 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	ctx := context.Background()
 
+	// Extract opentracing etc baggage from headers
+	// Annotate the inbound context with a trace span
+	tracer := h.Deps.Tracer()
+	carrier := opentracing.HTTPHeadersCarrier(req.Header)
+	parentSpanCtx, _ := tracer.Extract(opentracing.HTTPHeaders, carrier)
+	span := tracer.StartSpan(
+		treq.Procedure,
+		opentracing.StartTime(start),
+		opentracing.Tags{
```

Comment:
Perhaps we should submit `rpc.transport`? Would be nice to know which transport was issuing the req.